### PR TITLE
test sign in for name, email, password

### PR DESCRIPTION
**Pull request by kriizjones**
Pivotal Tracker story : https://www.pivotaltracker.com/n/projects/1935691

Changes for this pull request 👍 

Changed and fixed some stuff that where some comments. 
Fixed lines etc. 

Create cucumber test for the signup feature

### CODE REVIEW

**Comment by AmberWilkie** [comment]
I'd like to see more commits, Kristoffer. You should commit when you make a test pass.

File: features/signup.feature [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      And I fill in "Password confirmation" with "password"
+      And I click "Create"
+      Then I should see "Email can't be blank"
+
+  Scenario: User puts no password
+    When I fill in "Name" with "anna"
+    And I fill in "Email" with "anna@msn.com"
+    And I fill in "Password confirmation" with "password"
+    And I click "Create"
+    Then I should see "Password can't be blankPassword"
```
Comment (AmberWilkie) [inline features/signup.feature]: Why is there a password on the end of this line?

**Review by AmberWilkie — COMMENTED** [review]
Looks good - just a tiny thing to fix up and it's ready to merge.

File: features/signup.feature [hunk 2]
```diff
@@ -0,0 +1,42 @@
+Feature: Sign up
+
```
Comment (tochman) [inline features/signup.feature:2]: User story please

File: features/signup.feature [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  Scenario: Sign up to be a user
+    When I fill in "Name" with "anna"
+    And I fill in "Email" with "anna@msn.com"
+    And I fill in "Password" with "password"
+    And I fill in "Password confirmation" with "password"
+    And I click "Create"
+    Then I should see "Welcome! You have signed up successfully."
+
+
```
Comment (tochman) [inline features/signup.feature]: One line break is enough

File: features/signup.feature [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    Scenario: User puts no email
+      When I fill in "Name" with "anna"
+      And I fill in "Password" with "password"
+      And I fill in "Password confirmation" with "password"
+      And I click "Create"
+      Then I should see "Email can't be blank"
```
Comment (tochman) [inline features/signup.feature]: Be mindful with code indentation

File: features/step_definitions/signup_steps.rb [hunk 5]
```diff
@@ -0,0 +1,2 @@
+
+
```
Comment (tochman) [inline features/step_definitions/signup_steps.rb]: Why is this file empty?

**Review by tochman — CHANGES_REQUESTED** [review]

File: features/signup.feature (hunk 1 above)
Comment (kriizjones) [reply features/signup.feature]: fixed

File: features/signup.feature (hunk 2 above)
Comment (kriizjones) [reply features/signup.feature:2]: fixed

File: features/signup.feature (hunk 3 above)
Comment (kriizjones) [reply features/signup.feature]: fixed

File: features/step_definitions/signup_steps.rb (hunk 5 above)
Comment (kriizjones) [reply features/step_definitions/signup_steps.rb]: fixed.

File: features/signup.feature (hunk 4 above)
Comment (kriizjones) [reply features/signup.feature]: fixed

**Comment by tochman** [comment]
I see no PR description.

**Review by tochman — APPROVED** [review]

**Comment by tochman** [comment]
This will fail since there is no travis.yml file. Thing need to happen in the right sequence.

**Comment by tochman** [comment]
```
Changed and fixed some stuff that where some comments.
Fixed lines etc.
```
Is *not* a good PR description.

File: .travis.yml [hunk 6]
```diff
@@ -3,6 +3,7 @@ cache: bundler
 rvm:
   - 2.2.1
 script:
+  - "bundle exec rake neo4j:install['community-2.2.0-M02'] neo4j:start default --trace"
```
Comment (tochman) [inline .travis.yml:6]: ?? What is this line doing?

**Comment by tochman** [comment]
Ping @AmberWilkie and @kriizjones Plz see my comments.

File: .travis.yml (hunk 6 above)
Comment (kriizjones) [reply .travis.yml:6]: trying out some lines but have delete them.

**Pull request merged**